frontend/packages/console-plugin-sdk/src/useExtensions.ts

```diff
@@ -63,4 +63,4 @@ export const useExtensions: UseExtensions = (typeGuard) => {
   return extensionsInUse;
 };
 
-type UseExtensions = <E extends Extension>(typeGuard: ExtensionTypeGuard<E>) => readonly E[];
```

Comment:
See https://github.com/openshift/console/pull/4054#discussion_r370231111 for details on this change.